tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -3,14 +3,14 @@
 import io.opentracing.Tracer;
 import org.axonframework.commandhandling.CommandMessage;
 import org.axonframework.eventhandling.DomainEventMessage;
-import org.axonframework.eventhandling.EventMessage;
 import org.axonframework.messaging.Message;
 import org.axonframework.queryhandling.QueryMessage;
 
 /**
  * Utility class providing methods useful for attaching information to Spans
  */
```

Comment:
Nit: Adding the `since` and `author` tags would be nice too whilst you're at it.

Reply:
Since the one I really touched is `TracingCommandGateway`, I will add it there.